File: datasets/java/valid/Valid0491.java
public class Valid0491 {
    private int value;
    
    public Valid0491(int value) {
        this.value = value;
    }
    
    public int getValue() {
        return value;
    }
    
    public static void main(String[] args) {
        Valid0491 obj = new Valid0491(42);
        System.out.println("Value: " + obj.getValue());
    }
}

File: datasets/python/valid/valid_0491.py
class Handler:
    def __init__(self, res):
        self._res = res

    def get_res(self):
        return self._res

    def set_res(self, item):
        self._res = item

obj = Handler(16)
print(obj.get_res())
obj.set_res(25)
print(obj.get_res())
